tests/cases/standalone/common/promql/label.result

```diff
@@ -0,0 +1,103 @@
+CREATE TABLE test (
+  ts timestamp(3) time index,
+  host STRING,
+  idc STRING,
+  val BIGINT,
+  PRIMARY KEY(host, idc),
+);
+
+Affected Rows: 0
+
+INSERT INTO TABLE test VALUES
+    (0,     'host1', 'idc1', 1),
+    (0,     'host2', 'idc1', 2),
+    (5000,  'host1', 'idc2:zone1',3),
+    (5000,  'host2', 'idc2',4),
+    (10000, 'host1', 'idc3:zone2',5),
+    (10000, 'host2', 'idc3',6),
+    (15000, 'host1', 'idc4:zone3',7),
+    (15000, 'host2', 'idc4',8);
+
+Affected Rows: 8
+
+-- Missing source labels --
+TQL EVAL (0, 15, '5s') label_join(test{host="host1"}, "new_host", "-");
+
+Error: 1004(InvalidArguments), Invalid function argument for label_join
+
+-- SQLNESS SORT_RESULT 3 1
+TQL EVAL (0, 15, '5s') label_join(test{host="host1"}, "new_host", "-", "idc", "host");
+
++---------------------+-----+------------------+-------+------------+
+| ts                  | val | new_host         | host  | idc        |
++---------------------+-----+------------------+-------+------------+
+| 1970-01-01T00:00:00 | 1   | idc1-host1       | host1 | idc1       |
+| 1970-01-01T00:00:05 | 1   | idc1-host1       | host1 | idc1       |
+| 1970-01-01T00:00:05 | 3   | idc2:zone1-host1 | host1 | idc2:zone1 |
+| 1970-01-01T00:00:10 | 1   | idc1-host1       | host1 | idc1       |
+| 1970-01-01T00:00:10 | 3   | idc2:zone1-host1 | host1 | idc2:zone1 |
+| 1970-01-01T00:00:10 | 5   | idc3:zone2-host1 | host1 | idc3:zone2 |
+| 1970-01-01T00:00:15 | 1   | idc1-host1       | host1 | idc1       |
+| 1970-01-01T00:00:15 | 3   | idc2:zone1-host1 | host1 | idc2:zone1 |
+| 1970-01-01T00:00:15 | 5   | idc3:zone2-host1 | host1 | idc3:zone2 |
+| 1970-01-01T00:00:15 | 7   | idc4:zone3-host1 | host1 | idc4:zone3 |
```

Comment:
I'm not sure if this is valid from the output order. values within the same time-series (after label_join) are not continuous. Could you add a test like rate with label_join to validate it?

Reply:
Confirm that Prometheus also does not support the range selector for these function results, so it is irrelevant.